skunkworks_crow/src/main/java/org/odk/share/views/ui/send/fragment/FilledFormsFragment.java

```diff
@@ -150,13 +155,40 @@ private void setEmptyViewVisibility(int len) {
 
     @OnClick(R.id.send_button)
     public void send() {
-        Intent intent = new Intent(getActivity(), SendActivity.class);
+        Intent hotspotIntent = new Intent(getActivity(), HpSenderActivity.class);
```

Comment:
Create intent in the switch block itself and only one intent is required because there can only be one option either hotspot or bluetooth.

Reply:
Fixed.